buildres/linux/postinst

```diff
@@ -20,9 +20,8 @@ set -e
 case "$1" in
     configure)
         INSTALL_PATH="/usr/lib/mozilla/native-messaging-hosts/"
-        mkdir -p $INSTALL_PATH
-        \cp /opt/jabref/lib/org.jabref.jabref.json $INSTALL_PATH
-        chmod a+x "/opt/jabref/lib/jabrefHost.py"
+        install -D -m0755 /opt/jabref/lib/org.jabref.jabref.json $INSTALL_PATH
+        install -D -m0644 /opt/jabref/lib/jabref.desktop /usr/share/applications/
```

Comment:
When I created the postinst file, I made a mistake and didn't used the template from https://hg.openjdk.java.net/jdk/sandbox/file/47f0d21c7e8d/src/jdk.jpackage/linux/classes/jdk/jpackage/internal/resources/template.postinst. Could you please use this as a base. Should make it unnecessary to copy the desktop file by hand.

Reply:
The page is not loading :(

Reply:
Same here...stange

Reply:
java.net was down today, is working again